net-libs/nativebiginteger/nativebiginteger-2.1.0.ebuild

```diff
@@ -17,16 +17,20 @@ RESTRICT="!test? ( test )"
 
 DEPEND="
 	dev-libs/gmp:0=
-	virtual/jdk:1.8
+	|| (
+		virtual/jdk:11
+		virtual/jdk:17
+	)
+"
+RDEPEND="
+	|| (
+		virtual/jre:11
+		virtual/jre:17
+	)
```

Comment:
Can't this be simplified with 
```
	>=virtual/jdk-11:*
```
and
```
	>=virtual/jre-11:*
```
See https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE

Reply:
Indeed, that'll be easier to keep up with newer jdks.